Task5-2/webpack.config.js

```diff
@@ -0,0 +1,52 @@
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const CopyPlugin = require('copy-webpack-plugin');
+
+module.exports = {
```

Comment:
Some additional tasks:
- configure autoprefixer (using postcss)
- after you fix other comments in this file (as result, all css code will be located inside single `main.css`), add CSS minification and add sourcemaps

Reply:
Done